### PR TITLE
Add NPCs from PFS 4-15

### PR DESCRIPTION
**Pull request by TMunsie**
Also correct strike value for lesser animate dream from PFS 3-19

### CODE REVIEW

File: packs/pfs-season-4-bestiary/elite-smokesight-wight-commander.json [hunk 1]
```diff
@@ -0,0 +1,1306 @@
...
+            "system": {
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "interaction",
+                "description": {
+                    "value": "<p>The wight commander ignores the @UUID[Compendium.pf2e.conditionitems.DmAIPqOBomZ7H95W]{Concealed} condition from smoke.</p>"
```
Comment (simonward) [inline packs/pfs-season-4-bestiary/elite-smokesight-wight-commander.json:731]: There are a few old format UUID compendium links. Might be easiest to run Shark's script after this is merged?

Comment (TikaelSol) [reply packs/pfs-season-4-bestiary/elite-smokesight-wight-commander.json:731]: We should probably fix extract.

Comment (stwlam) [reply packs/pfs-season-4-bestiary/elite-smokesight-wight-commander.json:731]: If I fix extract, it'll be to throw an error on malformed UUIDs.

Comment (TMunsie) [reply packs/pfs-season-4-bestiary/elite-smokesight-wight-commander.json:731]: These were all generated from the base monsters in Book of the Dead

Comment (stwlam) [reply packs/pfs-season-4-bestiary/elite-smokesight-wight-commander.json:731]: From V10?

File: packs/pfs-season-4-bestiary/elite-clacking-skull-swarm.json [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+            "system": {
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 2
+                },
+                "category": "offensive",
+                "description": {
+                    "value": "<p>The clacking skull swarm emits a terrifying, painful scream that deals [[/r 10d6+4[sonic]]] damage to all creatures in a @Template[type:cone|distance:30] (@Check[type:will|dc:31|basic:true] save). A creature that fails this save is also @UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 1} (@UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 2} on a critical failure).</p>\n<p>The swarm can't use Frightening Scream again for [[/br 1d4 #Recharge Shrieking Scream]]{1d4 rounds}.</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/elite-clacking-skull-swarm.json:219]:
```suggestion
                    "value": "<p>The clacking skull swarm emits a terrifying, painful scream that deals [[/r (10d6+4)[sonic]]] damage to all creatures in a @Template[type:cone|distance:30] (@Check[type:will|dc:31|basic:true] save). A creature that fails this save is also @UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 1} (@UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 2} on a critical failure).</p>\n<p>The swarm can't use Frightening Scream again for [[/br 1d4 #Recharge Shrieking Scream]]{1d4 rounds}.</p>"
```

File: packs/pfs-season-4-bestiary/elite-clacking-skull-swarm.json [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+            "system": {
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 1
+                },
+                "category": "offensive",
+                "description": {
+                    "value": "<p>Each enemy in the swarm's space takes [[/r 4d8+2[piercing]]] damage (@Check[type:reflex|dc:31|basic:true] save).</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/elite-clacking-skull-swarm.json:262]:
```suggestion
                    "value": "<p>Each enemy in the swarm's space takes [[/r (4d8+2)[piercing]]] damage (@Check[type:reflex|dc:31|basic:true] save).</p>"
```

File: packs/pfs-season-4-bestiary/elite-clacking-skull-swarm.json [hunk 4]
```diff
@@ -0,0 +1,470 @@
...
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "defensive",
+                "deathNote": true,
+                "description": {
+                    "value": "<p>When a clacking skull swarm is reduced to 0 Hit Points, it erupts in an explosion of foul energy and bone fragments in a @Template[type:burst|distance:30], dealing [[/r {2d12+2[piercing],2d12+2[negative]}]]{2d12+2 piercing damage and 2d12+2 negative damage} (@Check[type:reflex|dc:31|basic:true] save).</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/elite-clacking-skull-swarm.json:112]:
```suggestion
                    "value": "<p>When a clacking skull swarm is reduced to 0 Hit Points, it erupts in an explosion of foul energy and bone fragments in a @Template[type:burst|distance:30], dealing [[/r (2d12+2)[piercing],(2d12+2)[negative]]]{2d12+2 piercing damage and 2d12+2 negative damage} (@Check[type:reflex|dc:31|basic:true] save).</p>"
```

File: packs/pfs-season-4-bestiary/elite-combustive-zombie.json [hunk 5]
```diff
@@ -0,0 +1,479 @@
...
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "defensive",
+                "deathNote": true,
+                "description": {
+                    "value": "<p>When a combustive zombie dies, its body explodes in a @Template[type:burst|distance:30] of fire and debris that deals [[/r {3d10+2[bludgeoning],3d10+2[fire]}]]{3d10 + 2 bludgeoning damage and 3d10 + 2 fire damage} to each creature in the area (@Check[type:reflex|dc:26|basic:true] save).</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/elite-combustive-zombie.json:233]:
```suggestion
                    "value": "<p>When a combustive zombie dies, its body explodes in a @Template[type:burst|distance:30] of fire and debris that deals [[/r (3d10+2)[bludgeoning],(3d10+2)[fire]]]{3d10 + 2 bludgeoning damage and 3d10 + 2 fire damage} to each creature in the area (@Check[type:reflex|dc:26|basic:true] save).</p>"
```

File: packs/pfs-season-4-bestiary/elite-smokesight-deadeye-wight.json [hunk 6]
```diff
@@ -0,0 +1,839 @@
...
+                "actionType": {
+                    "value": "reaction"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "defensive",
+                "deathNote": true,
+                "description": {
+                    "value": "<p><strong>Trigger</strong> The deadeye wight is reduced to 0 Hit Points</p>\n<hr />\n<p><strong>Effect</strong> The deadeye wight has rigged their own body with a snare, which they trigger before being destroyed. Slivers of bone launch at an adjacent creature, dealing [[/r 4d8+4[piercing]]] damage (@Check[type:reflex|dc:27|basic:true]).</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/elite-smokesight-deadeye-wight.json:530]:
```suggestion
                    "value": "<p><strong>Trigger</strong> The deadeye wight is reduced to 0 Hit Points</p>\n<hr />\n<p><strong>Effect</strong> The deadeye wight has rigged their own body with a snare, which they trigger before being destroyed. Slivers of bone launch at an adjacent creature, dealing [[/r (4d8+4)[piercing]]] damage (@Check[type:reflex|dc:27|basic:true]).</p>"
```

File: packs/pfs-season-4-bestiary/elite-smokesight-hunter-wight.json [hunk 7]
```diff
@@ -0,0 +1,839 @@
...
+                "actionType": {
+                    "value": "reaction"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "defensive",
+                "deathNote": true,
+                "description": {
+                    "value": "<p><strong>Trigger</strong> The hunter wight is reduced to 0 Hit Points</p>\n<hr />\n<p><strong>Effect</strong> The hunter wight has rigged their own body with a snare, which they trigger before being destroyed. Slivers of bone launch at an adjacent creature, dealing [[/r 4d8+4[piercing]]] damage (@Check[type:reflex|dc:27|basic:true]).</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/elite-smokesight-hunter-wight.json:530]:
```suggestion
                    "value": "<p><strong>Trigger</strong> The hunter wight is reduced to 0 Hit Points</p>\n<hr />\n<p><strong>Effect</strong> The hunter wight has rigged their own body with a snare, which they trigger before being destroyed. Slivers of bone launch at an adjacent creature, dealing [[/r (4d8+4)[piercing]]] damage (@Check[type:reflex|dc:27|basic:true]).</p>"
```

File: packs/pfs-season-4-bestiary/elite-sulfur-zombie.json [hunk 8]
```diff
@@ -0,0 +1,478 @@
...
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "defensive",
+                "deathNote": true,
+                "description": {
+                    "value": "<p>When a sulfur zombie dies, its body explodes in a @Template[type:burst|distance:30] of fire and debris that deals [[/r {2d10+2[bludgeoning],2d10+2[fire]}]]{2d10+2 bludgeoning damage and 2d10+2 fire damage} to each creature in the area (@Check[type:reflex|dc:23|basic:true] save).</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/elite-sulfur-zombie.json:233]:
```suggestion
                    "value": "<p>When a sulfur zombie dies, its body explodes in a @Template[type:burst|distance:30] of fire and debris that deals [[/r (2d10+2)[bludgeoning],(2d10+2)[fire]]]{2d10+2 bludgeoning damage and 2d10+2 fire damage} to each creature in the area (@Check[type:reflex|dc:23|basic:true] save).</p>"
```

File: packs/pfs-season-4-bestiary/weak-clacking-skull-swarm.json [hunk 9]
```diff
@@ -0,0 +1,470 @@
...
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "defensive",
+                "deathNote": true,
+                "description": {
+                    "value": "<p>When a clacking skull swarm is reduced to 0 Hit Points, it erupts in an explosion of foul energy and bone fragments in a @Template[type:burst|distance:30], dealing [[/r {2d12-2[piercing],2d12-2[negative]}]]{2d12-2 piercing damage and 2d12-2 negative damage} (@Check[type:reflex|dc:27|basic:true] save).</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/weak-clacking-skull-swarm.json:112]:
```suggestion
                    "value": "<p>When a clacking skull swarm is reduced to 0 Hit Points, it erupts in an explosion of foul energy and bone fragments in a @Template[type:burst|distance:30], dealing [[/r (2d12-2)[piercing],(2d12-2)[negative]]]{2d12-2 piercing damage and 2d12-2 negative damage} (@Check[type:reflex|dc:27|basic:true] save).</p>"
```

File: packs/pfs-season-4-bestiary/weak-clacking-skull-swarm.json [hunk 10]
```diff
@@ -0,0 +1,470 @@
...
+            "system": {
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 1
+                },
+                "category": "offensive",
+                "description": {
+                    "value": "<p>Each enemy in the swarm's space takes [[/r 4d8-2[piercing]]] damage (@Check[type:reflex|dc:27|basic:true] save).</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/weak-clacking-skull-swarm.json:262]:
```suggestion
                    "value": "<p>Each enemy in the swarm's space takes [[/r (4d8-2)[piercing]]] damage (@Check[type:reflex|dc:27|basic:true] save).</p>"
```

File: packs/pfs-season-4-bestiary/weak-clacking-skull-swarm.json [hunk 11]
```diff
@@ -0,0 +1,470 @@
...
+            "system": {
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "defensive",
+                "description": {
+                    "value": "<p>@Template[type:emanation|distance:60]{60 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Aura]</p>\n<hr />\n<p>A clacking skull swarm emits a cacophony of chattering. A creature entering or beginning its turn within the area must succeed at a @Check[type:will|dc:27] save or become @UUID[Compendium.pf2e.conditionitems.Item.Confused] for 1 round. A creature that successfully saves is immune to that swarm's chattering teeth for 24 hours. The swarm can stop or resume this ability as a free action.</p>"
```
Comment (TikaelSol) [inline packs/pfs-season-4-bestiary/weak-clacking-skull-swarm.json:149]:
```suggestion
                    "value": "<p>60 feet. A clacking skull swarm emits a cacophony of chattering. A creature entering or beginning its turn within the area must succeed at a @Check[type:will|dc:27] save or become @UUID[Compendium.pf2e.conditionitems.Item.Confused] for 1 round. A creature that successfully saves is immune to that swarm's chattering teeth for 24 hours. The swarm can stop or resume this ability as a free action.</p>"
```

**Review by TikaelSol — APPROVED** [review]

**Pull request merged**